### ISSUE TITLE
Metamask mobile: issues when number pad is open

### ISSUE THREAD

**Issue by matt-bullock** [AugurProject/augur#10853]
Top bar gets hidden and there's a gap above keyboard

![image](https://user-images.githubusercontent.com/39496225/110985878-43f90800-836d-11eb-8d91-534485b2abd3.png)

**Comment by andrewdamelio** [comment]
@matt-bullock do you still see this issue?

If yes, could you also check and confirm if you see the same issue on uniswap when entering a value for swap amount.

**Comment by matt-bullock** [comment]
@andrewdamelio yes still seeing the issue. No issues on uni but you'll notice they have their top bar above the keyboard so if you're forking something from uni then this might be why

**Comment by Chwy5** [comment]
@andrewdamelio what is status on this
